Doc/library/platform.rst

```diff
@@ -301,3 +308,39 @@ Linux Platforms
           return ids
 
    .. versionadded:: 3.10
+
+
+Android Platform
+----------------
+
+.. function:: android_ver(release="", api_level=0, min_api_level=0, \
+                          manufacturer="", model="", device="")
+
+   Get Android device information. Returns a :func:`~collections.namedtuple`
+   with the following attributes. Values which cannot be determined are set to
+   the defaults given as parameters.
+
+   * ``release`` - Android version, as a string (e.g. ``"14"``)
+
+   * ``api_level`` - API level, as an integer (e.g. ``34``)
+
+   * ``min_api_level`` - Minimum API level this build of Python can run on, as
+     an integer (e.g. ``21``)
+
+   * ``manufacturer`` - `manufacturer name
+     <https://developer.android.com/reference/android/os/Build#MANUFACTURER>`__
+     (e.g. ``"Google"``)
+
+   * ``model`` - `model name
+     <https://developer.android.com/reference/android/os/Build#MODEL>`__
+     (e.g. ``"Pixel 7"``)
+
+   * ``device`` - `device name
+     <https://developer.android.com/reference/android/os/Build#DEVICE>`__
+     (e.g. ``"panther"``)
+
+   Which one of ``model`` and ``device`` is more likely to distinguish different
```

Comment:
@mhsmith "Which one of" is a bit unclear to me. Perhaps this could be rewritten to be a bit more explicit.

Maybe: Manufacturers use ``model`` and ``device`` to identify Android hardware. Specific use of these two names will vary by manufacturer to identify the marketing name of the hardware.

Reply:
OK, I've rephrased it to describe the typical values, and added a link to a list of known devices.

Reply:
I've also reverted the documentation changes in `platform.system` and `platform.release` to avoid a conflict with #117052, which documents those functions for both Android and iOS.